src/core/internal/fetch/fetchClient.ts

```diff
@@ -1,7 +1,12 @@
 import { HttpClient } from '../..';
+import { RequestOptions } from 'http';
```

Comment:
We need to avoid such imports from Node.js modules.

Reply:
Proposal. Create interface near HttpClient interface with headers and cors flag and use it as a type of config in HttpClient.

Reply:
Ohhh... yeah, I forgot it